[Merged by Bors] - Add page for CleanCut

CleanCut opened the pull request:
☝🏻 

This is an awesome idea.

cart commented:
bors r+

mockersf commented on Org Members/CleanCut.toml:
```diff
@@ -0,0 +1,50 @@
+# Optional
+github = "CleanCut"
+
+discord = "DiscordUsername#1234"
+# This allows direct linking to your profile for people with a shared server
+# https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-
+# Optional
+discord-userid = ""
```
@CleanCut are you sure about your Discord user? 😄

CleanCut commented:
@mockersf 🤦🏻‍♂️ I noticed, fixed it, committed it...and forgot to push the commit. 😝 

Fixed in https://github.com/bevyengine/bevy-community/pull/28
